### DISCUSSION TITLE
[@types/jsrsasign] Add missing function to modules/KJUR/crypto/ECDSA.d.ts

### DISCUSSION THREAD

**Discussion by irririki** [DefinitelyTyped/DefinitelyTyped#62333] [Issues with a @types package]
Add function definition to class ECDSA as follows:

/**
     * generate public key for EC private key
     * @name generatePublicKeyHex
     * @memberOf KJUR.crypto.ECDSA#
     * @function
     * @return {String} associative array of hexadecimal string of private and public key
     * @example
     * var ec = new KJUR.crypto.ECDSA({'curve': 'secp256r1', 'prv': prvHex});
     * var pubhex = ec.generatePublicKeyHex(); // hexadecimal string of EC public key
     * var pub ec.getPublicKeyXYHex() &rarr; { x: '01bacf...', y: 'c3bc22...' }
     */
`
generatePublicKeyHex(): string;
`

[a comment by an automated account is not shown]

**Reply by ffflorian** [reply]
Hi @irririki, I'd be happy if you could create a PR with your mentioned changes :)

**Reply by irririki** [reply]
Done. See https://github.com/DefinitelyTyped/DefinitelyTyped/pull/62337
